setup.py

```diff
@@ -39,6 +39,7 @@
         'taca.subcommands': [
             'storage = taca.storage.cli:storage',
             'analysis = taca.analysis.cli:analysis',
+            'delivery = taca.utils.cli:delivery',
```

Comment:
Monitor? That's entirely new :grinning:

The data is in genstat under the bioinfo tab and the deliveries page. How about `bioinfo-delivery`?

See also above in the outdated comment diff.